extensions/amp-font/amp-font.md

```diff
@@ -16,8 +16,50 @@ limitations under the License.
 
 ### <a name="`amp-font`"></a> `amp-font`
 
-The `amp-font` extension can trigger and monitor the loading of custom fonts on AMP pages.
-
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Trigger and monitor the loading of custom fonts on AMP pages.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-font" src="https://cdn.ampproject.org/v0/amp-font-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><a href="https://github.com/ampproject/amphtml/blob/master/examples/font.amp.html">font.amp.html</a></td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-font` tag
+(see also `amp-font` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
+
+<table>
+  <tr>
+    <th width="40%"><strong>Validation Error</strong></th>
```

Comment:
optional. Unsure if worth mentioning, but the timeout attribute value must be a positive integer or you'll get INVALID_ATTR_VALUE